#request
bool use_gimbal
bool use_stereo
---
#response
bool result